application/helpers/expressions/em_manager_helper.php

```diff
@@ -8275,36 +8275,27 @@ private function getQuestionAttributesForEM($surveyid = 0, $qid = 0, $lang = '')
         if ($qid) {
             $oQids = Question::model()->findAll(
                 [
-                    'select'    => 'qid',
-                    'group'     => 'qid',
-                    'distinct'  => true,
                     'condition' => "qid=:qid and parent_qid=0",
```

Comment:
We better understand why the name it's $oQids and not $oQuestion : i think it's a good idea to rename this variable here.

Reply:
Thank you